{"name": "困人事件标记误报", "status": "passed", "attachments": [{"name": "log", "source": "fc942460-5a86-4c78-9dec-fdf596b64bb7-attachment.txt", "type": "text/plain"}], "start": 1690253017532, "stop": 1690253018010, "uuid": "942870f6-47f9-42f7-a243-ee1ded172531", "historyId": "ab566412ce2c9ed8e210215e4ae8a5dd", "testCaseId": "16b1c3c86243e1b02c591065644e1507", "fullName": "opsPC.test_emergency#test_update_deal_status_false_alarm_001", "labels": [{"name": "feature", "value": "运维管理平台"}, {"name": "story", "value": "应急处置"}, {"name": "parentSuite", "value": "opsPC"}, {"name": "suite", "value": "test_emergency"}, {"name": "host", "value": "cale"}, {"name": "thread", "value": "18116-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "opsPC.test_emergency"}]}